.gitignore

```diff
@@ -33,4 +33,5 @@ build/
 .vscode/
 
 ### Node modules###
-node_modules/
\ No newline at end of file
+node_modules/
+/src/test/java/system_tests/chromedriver*
```

Comment:
Why not commit the chrome driver to version control, is it for chrome version compatibility issues?

Reply:
It’s OS dependent.

Reply:
We should be able to use different file names though, no? I saw that we already have code that checks to OS an loads the right chromedriver.

Reply:
Also you got to ensure that the chromedriver matches the chrome version installed on your system. I don’t think chromedriver is something you would want to version control as it’s user specific.